controllers/serviceclaim_controller.go

```diff
@@ -116,12 +119,9 @@ func (r *ServiceClaimReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		sclaim.Status.ClaimID = uuid.New().String()
 		l.Info("reconciling new service claim")
 		return ctrl.Result{}, r.processPendingClaim(ctx, req, sclaim)
-	case primazaiov1alpha1.ServiceClaimStatePending:
-		l.Info("reconciling pending service claim")
-		return ctrl.Result{}, r.processPendingClaim(ctx, req, sclaim)
 	default:
-		l.Info("reconciling resolved service claim")
-		return ctrl.Result{}, nil
+		l.Info("reconciling resolved and pending service claim")
+		return ctrl.Result{}, r.processPendingClaim(ctx, req, sclaim)
```

Comment:
there is a mismatch between log line and function name. Are we processing resolved and pending or just pending claims?

Reply:
We are resolving both resolved as well as pending service claims here. For a resolved claim if the registered service is updated, i am using this method to extract the sed. Makes sense to rename it though, just `processClaim`